src/main/java/org/broadleafcommerce/vendor/amazon/s3/S3FileServiceProvider.java

```diff
@@ -306,4 +386,113 @@ public void setBroadleafFileService(BroadleafFileService bfs) {
     public void setConcurrentFileOutputStream(ConcurrentFileOutputStream concurrentFileOutputStream) {
         this.concurrentFileOutputStream = concurrentFileOutputStream;
     }
+    
+    @Override
+	public File getResource(String name, FileApplicationType fileApplicationType) {
+		File file = getResourceDefault(name, fileApplicationType);
+		if (file.getPath().contains("this/path/should/not/exist/")  && isMultiTenantEnvironment()) {
```

Comment:
Don't use this hardcoded string. Instead check `file.exists()`

Reply:
This came from the original code, the original method was returning this in this way, I know does not look good but I'm not sure if it is is being used outside this module.
I think cannot use file.exists() because this file may no exists in the file system.